orchestrator/api/api_v1/endpoints/processes.py

```diff
@@ -115,6 +120,7 @@ def delete(process_id: UUID) -> None:
     if not process:
         raise_status(HTTPStatus.NOT_FOUND)
 
+    broadcast_invalidate_status_counts()
```

Comment:
I wonder why the old broadcast statement was done above the database commit. Do you have any idea perhaps @tjeerddie 

I think we should move them to below the commit, to ensure that the broadcasting action can never delay the (more important) DB action, and eliminate the (very small) possibility that the UI has done something on the broadcasted event _before_ the DB action completed.

Reply:
(This would apply to all other broadcast statements as well)

Reply:
yea, haven't reviewed previous MR's strict enough, I agree that they should happen after DB actions